og.module

```diff
@@ -44,6 +44,24 @@ function og_entity_insert(EntityInterface $entity) {
 }
 
 /**
+ * Implements hook_entity_delete().
+ */
+function og_entity_delete(EntityInterface $entity) {
```

Comment:
Indeed! We also have `hook_entity_predelete()` now in Drupal 8, that is more appropriate for this.